website/archiver/utils.py

```diff
@@ -111,8 +111,9 @@ def handle_archive_fail(reason, src, dst, user, result):
         pass
     else:  # reason == ARCHIVER_UNCAUGHT_ERROR
         send_archiver_uncaught_error_mails(src, user, result, url)
-    dst.root.sanction.forcibly_reject()
-    dst.root.sanction.save()
+    if dst.root.sanction:
+        dst.root.sanction.forcibly_reject()
+        dst.root.sanction.save()
```

Comment:
This fixes a test where `dst.root.sanction` is None. However, from how the test is set-up, it shouldn't be `None`. Nevertheless, it's good to having an extra check.